### PR TITLE
option for plain ol' json

### LINKED ISSUE #5: Dyno.deserialize doesn't like non-wire-format JSON
The change I made in https://github.com/mapbox/dynamodb-migrator/pull/3 (use Dyno.deserialize, expecting wire-format from a backup instead of line-deliminated JSON) doesn't play nice with feeding in a list of keys that look like `{ id: "foo", collection: "foo:bar"}`.

I think the migrator should continue support for line-deliminated JSON, especially in stream mode (i.e. for  "give me a list of keys to target, and I'll do stuff to them").  Maybe this should be an additional option passed only in stream mode, like a `--json` flag or something. Since I messed this up in the first place I can take this on, let me know what you think @rclark.

### PR DESCRIPTION
**Pull request by emilymcafee**
fixes #5 

@rclark  :+1: for this approach? defaults to dynamo wire format unless `--json` flag is passed. open to renaming :)

going to add another test that actually uses the new flag.

### CODE REVIEW

**Comment by rclark** [comment]
Call it `--dyno`? I'm imagining documentation:

> Pass the `--dyno` flag to the migrator if your input JSON objects are in a format suitable for direct usage in [dyno](https://github.com/mapbox/dyno). Otherwise, it is assumed that the objects are formatted using standard DynamoDB syntax.

or something

File: index.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ var Dyno = require('dyno');
 var split = require('split');
 var Readable = require('stream').Readable;
 
-module.exports = function(method, database, migrate, stream, live, concurrency, callback) {
+module.exports = function(method, database, migrate, stream, live, json, concurrency, callback) {
```
Comment (rclark) [inline index.js]: parameter overload!!! @____________________@

That said, I think this is what we should do.

**Comment by emilymcafee** [comment]
https://github.com/mapbox/dynamodb-migrator/blob/json/index.js#L37

If we call it `--dyno`, will there be a namespace collision here?

File: index.js (hunk 1 above)
Comment (emilymcafee) [inline index.js]: ._.

yeah, but at least the user doesn't have to sort through constructing this function in the proper order.

**Comment by rclark** [comment]
oooooh names. Really not too important -- call it whatever! fwiw the flag could be `--dyno` even if this arg is called `expectPlainJaneJsonObectsDuringMigration`.

**Comment by emilymcafee** [comment]
Ohhh yes of course. Let's stick with `--dyno` then.

**Pull request merged**